plugins/mesh_flagger.js

```diff
@@ -1,178 +1,204 @@
-          highlighter.i = 0
-          highlighter.interval = setInterval(() => highlighter.flash(cubes, material), 1500)
-          highlighter.flash(cubes, material)
+    let aboutAction, button, button2, cubeAction
+    const id = "mesh_flagger"
+    const name = "Mesh Flagger"
+    const icon = "lightbulb"
+    const author = "SirJain and DerfX"
+    const links = {
+        TwitterSirJain: "https://twitter.com/SirJain2",
+        TwitterDerfX: "https://twitter.com/Derf31922027",
+        SirJainDiscord: "https://discord.gg/wM4CKTbFVN"
+    }
+
+    Plugin.register(id, {
+        title: name,
+        icon,
+        author,
+        description: "Flags meshes in a project on demand.",
+        about: "The **Generic model format** is the only format that supports meshes. If you convert to another format with meshes in your project, the meshes will disappear. You can choose to flag all meshes in the project through one click with this plugin. This lets you know what you are losing by converting your project, in case you looked over any.\n\n## How to use\nTo use this plugin, go to `File > Plugins > Available` and search for `Mesh Flagger`. Click install, then use `Tools > Flag Meshes` and click one of the options. Clicking the first option causes the plugin to flag all meshes in a project. The second option allows Blockbench to flag all meshes with six faces. This includes cuboid-meshes, but keep in mind it can flag non-cuboid meshes with six faces too!\n\n\nIt would be appreciated to report any bugs and suggestions!",
+        tags: ["Format: Generic Model", "Per-Face UV", "Meshes"],
+        version: "1.0.4",
+        min_version: "4.2.0",
+        variant: "both",
+        
+        oninstall() {
+            showAbout(true)
+            Blockbench.showQuickMessage("Successfully installed Mesh Flagger!", 3000)
+        },
+        onuninstall() {
+            Blockbench.showQuickMessage("Uninstalled Mesh Flagger", 3000)
         },
-        flash: (cubes, material) => {
-          var fc = highlighter.i
-          if (fc > 5) {
-            highlighter.running = false
-            clearInterval(highlighter.interval)
-          };
-          for (const cube of cubes) {
-            if (cube.mesh) {
-              cube.mesh.material = (fc % 2) ? material : cube.mesh.material_non_flash
+        onload() {
+            addAbout()
+            const highlighter = {
+                i: 0,
+                running: false,
+                start: (cubes, material) => {
+                    if (highlighter.running) return
+                    highlighter.running = true
+
+                    for (const cube of cubes) {
+                        cube.mesh.material_non_flash = cube.mesh.material
+                    };
+
+                    clearInterval(highlighter.interval)
+                    highlighter.i = 0
+                    highlighter.interval = setInterval(() => highlighter.flash(cubes, material), 1500)

```

Comment:
It's not the best user experience to have to wait for 1.5 seconds for something to happen after clicking. You should start in the flashed state. And maybe make the inverval generally faster.
Same applies for the cube flash plugin.

Reply:
I've fixed the empty flash at beginning part. My Utility Flaggers plugin which I will submit once this PR is merged will have customizable options for the duration, so I will leave that be for now.